fuel/downloaders/caltech101_silhouettes.py

```diff
@@ -0,0 +1,38 @@
+from fuel.downloaders.base import default_downloader
+
+
+BASE_URL = 'https://people.cs.umass.edu/~marlin/data/'
+FILENAME = 'caltech101_silhouettes_{}_split1.mat'
+
+
+def silhouettes_downloader(size, **kwargs):
+    if size not in (16, 28):
+        raise ValueError("size must be 16 or 28")
+
+    actual_filename = FILENAME.format(size)
+    actual_url = BASE_URL + actual_filename
+    return default_downloader(urls=[actual_url],
```

Comment:
Maybe it's a matter of personal preference, but `default_downloader` doesn't explicitly return anything, and having `silhouettes_downloader` return the `None` return value of `default_downloader` makes the function a little less readable IMHO.